cmake/tribits/core/installation/TribitsProjectConfigTemplate.cmake.in

```diff
@@ -124,8 +124,6 @@ FOREACH(comp ${PDOLLAR}{COMPONENTS_LIST})
    LIST(APPEND ${PROJECT_NAME}_TPL_LIBRARY_DIRS ${PDOLLAR}{${PDOLLAR}{comp}_TPL_LIBRARY_DIRS})
    LIST(APPEND ${PROJECT_NAME}_TPL_LIBRARIES ${PDOLLAR}{${PDOLLAR}{comp}_TPL_LIBRARIES})
  ELSE()
-   # Component not found.
-   MESSAGE(WARNING "Component \"${PDOLLAR}{comp}\" NOT found.")
    # Set ${PROJECT_NAME}_<component>_FOUND to FALSE.
    SET(${PROJECT_NAME}_${PDOLLAR}{comp}_FOUND FALSE)
```

Comment:
Is `PDOLLAR` defined by tribits?

Reply:
Yes, it is a special trick to avoid premature variable substitution.  It is defined at:

https://github.com/TriBITSPub/TriBITS/blob/master/tribits/core/package_arch/TribitsWriteClientExportFiles.cmake#L796

as:

```
SET(PDOLLAR "$")
```

This was put in by someone else a long time ago but this trick seems to work.  Note:

* https://cmake.org/cmake/help/v3.13/command/configure_file.html